GliaWidgets/View/Call/Video/VideoStreamView.swift

```diff
@@ -16,6 +16,7 @@ class VideoStreamView: View {
                 streamView.scale = .aspectFill
                 addSubview(streamView)
                 streamView.autoPinEdgesToSuperviewEdges()
+                streamView.layoutIfNeeded()
```

Comment:
Please correct me, but this setter looks for me as incorrect. I don't see but where is `streamView` set? If nowhere then line 13 is `true` always and line 14 do nothing...
Please fix this setter.

Reply:
`streamVideoView` is accessible from other classes, and this is exactly how it's done. `CallViewController` (lines 81-84) is responsible for setting that value when triggered by a ViewModel's event.

Reply:
this `setter` doesn't store value and 3 times calls `getter`. 
`getter` instead of returns value gets the first subview and casts it to the proper class. I don't see the reason for it and decided to ask you. For me, it looks pretty weird and makes places for bugs.

Reply:
I agree with @dukhovnyi

Reply:
How about:

`
    public weak var streamView: StreamView? {
        didSet {
            streamView.map { layout(oldStreamView: oldValue, streamView: $0) }
        }
    }

    private func layout(
        oldStreamView: StreamView?,
        streamView: StreamView
    ) {
        oldStreamView?.removeFromSuperview()
        streamView.scale = .aspectFill
        addSubview(streamView)
        streamView.autoPinEdgesToSuperviewEdges()
        streamView.layoutIfNeeded()
    }
`

Reply:
Damn, now I realize what you meant. Excuse my denseness. Will check/fix ASAP

Reply:
@kuldar-daniel great suggestion, but I made some changes to it. The intended functionality is to remove the old `streamView` from the superview if the new value is nil (which is exactly what happens when the camera is turned off).

Reply:
makes sense